Browser_IDE/executionEnvironment_Internal.js

```diff
@@ -398,7 +398,7 @@ window.addEventListener('message', function(m){
                 // If we got a syntax error from Babel, we know the browser can't return a more user friendly
                 // one since it didn't report one initially. So for now just report Unknown error.
                 // TODO: Report Babel's syntax error.
-                ReportError(userCodeBlockIdentifier+m.data.name, "Unknown syntax error.", null);
+                ReportError(userCodeBlockIdentifier+m.data.name, "Syntax error in code block: " + e.message, null);
```

Comment:
Just to chime in here, actually this is a good idea (and I actually did something similar in my branch 🤣 ) - I'm not sure why I decided not to report Babel's errors initially. They're less readable/friendly than the browser's, but still better to report what we can. 
Only suggestion would be to maybe report both `e.message` _and_ `e.name`, since `e.name` still tells the user the general type of error (e.g. 'SyntaxError'). Personally I just used `e.toString()`.
Also, if you want a test-case, the line `#include "splashkit.h"` gets through the browser's checking but makes Babel complain 😄